src/common/storage/collection_and_field_names.py

```diff
@@ -303,6 +303,8 @@
 
 ### samples
 
+SAMPLES_PRODUCT_ID = "samples"
```

Comment:
If we're going to define the product ID here we should also use this var in the samples data product

Reply:
yea. I am planning to update it accordingly in my next PRs when I am at `samples.py`.

Reply:
:+1: